bucket/azure-kubelogin.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "0.0.11",
+    "description": "A client-go credential (exec) plugin implementing azure authentication.",
+    "homepage": "https://github.com/Azure/kubelogin/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Azure/kubelogin/releases/download/v0.0.11/kubelogin-win-amd64.zip",
+            "hash": "8d509b64638bf2acba247b5333b8dc1f16a578ecb1917b562ec1ee938e49d85f"
+        }
+    },
+    "bin": "bin\\windows_amd64\\kubelogin.exe",
```

Comment:
This might conflict with https://github.com/ScoopInstaller/Extras/blob/cc0020bb0368d7380ebbef24c4cc8e214bfce0a4/bucket/kubelogin.json#L12

Reply:
The package name wont conflict, but the binary will. This isn't a Scoop problem, but a problem with vendors not checking for prior art. I strongly expect that these two tools are mutually exclusive. The binary needs to be called `kubelogin` so there is no way round it.